app/db/migrations/versions/1787d27cfda_.py

```diff
@@ -0,0 +1,33 @@
+# flake8: noqa
```

Comment:
Wouldn't it be better to create just one migration file? Or is there an idea behind three separate files?